### PR TITLE
DOCSP-42328: Specify a query

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-scala/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-42328
Staging - https://deploy-preview-57--docs-scala.netlify.app/read/specify-a-query/

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/read/specify-a-query.txt [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+To learn more about querying documents, see the :manual:`Query Documents
+</tutorial/query-documents/>` guide in the {+mdb-server+} manual.
+
+To learn more about retrieving documents with the {+driver-short+}, see the
+:ref:`scala-retrieve` guide.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods or types discussed in this
```
Comment (jordan-smith721) [inline source/read/specify-a-query.txt:364]:
```suggestion
To learn more about any of the methods discussed in this
```

**Review by jordan-smith721 — APPROVED** [review]
Nice work! LGTM

File: source/includes/read/specify-queries.scala [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    // Inserts documents representing fruits
+    val fruits: Seq[Document] = Seq(
+        Document("_id" -> 1, "name" -> "apples", "qty" -> 5, "rating" -> 3, "color" -> "red", "type" -> Seq("fuji", "honeycrisp")),
+        Document("_id" -> 2, "name" -> "bananas", "qty" -> 7, "rating" -> 4, "color" -> "yellow", "type" -> Seq("cavendish")),
+        Document("_id" -> 3, "name" -> "oranges", "qty" -> 6, "rating" -> 2, "type" -> Seq("naval", "mandarin")),
+        Document("_id" -> 4, "name" -> "pineapples", "qty" -> 3, "rating" -> 5, "color" -> "yellow")
+    )
+
+    collection.insertMany(fruits)
```
Comment (rozza) [inline source/includes/read/specify-queries.scala:26]: This doesn't do anything apart from create an `Observable` it must be subscribed to, to actually write some data.

**Review by rozza — CHANGES_REQUESTED** [review]

**Review by rozza — APPROVED** [review]
LGTM

**Pull request merged**